scipy/signal/filter_design.py

```diff
@@ -252,8 +254,8 @@ def freqs_zpk(z, p, k, worN=None):
     return w, h
 
 
-def freqz(b, a=1, worN=None, whole=False, plot=None):
-    """
+def freqz(b, a=1, worN=512, whole=False, plot=None, axis=0):
```

Comment:
Well, not everyone will agree with the reasoning, and even those who do probably wouldn't feel that the argument was strong enough to now go back and change all defaults to `None`.  I've only relatively recently started to prefer using `None` in my code.

Still, I'd rather we not move in the other direction of replacing `None`.  And in this case, it seems unrelated to the enhancement of this pull request.